doc/build/README.md

```diff
@@ -0,0 +1,25 @@
+<a name="logo"/>
+<div align="center">
+<a href="https://julialang.org/" target="_blank">
+<img src="https://julialang.org/images/logo_hires.png" alt="Julia Logo" width="210" height="142"></img>
+</a>
+</div>
+
+This directory contains various details related to building Julia:
+
+* [Detailed build instructions](https://github.com/JuliaLang/julia/blob/master/doc/build/build.md)
+
+Notes for various OSes:
+
+* [Linux](https://github.com/JuliaLang/julia/blob/master/doc/build/linux.md)
+* [macOS](https://github.com/JuliaLang/julia/blob/master/doc/build/macos.md)
+* [Windows](https://github.com/JuliaLang/julia/blob/master/doc/build/windows.md)
+* [FreeBSD](https://github.com/JuliaLang/julia/blob/master/doc/build/freebsd.md)
+
+Notes for various architectures:
+
+* [ARM](https://github.com/JuliaLang/julia/blob/master/doc/build/arm.md)
+
+Notes for building Julia for distribution:
+
+* [Distribution Notes](https://github.com/JuliaLang/julia/blob/master/doc/build/distributing.md)
```

Comment:
Isn't the name of this file uppercase?

Reply:
Nevermind, this was fixed.

Reply:
I hope OS X does not freak out with this case changing. It seemed ok on my computer.